### PR TITLE
EASY-1672: Make 'SF_PLAY_MODE' mandatory; EASY-1756: eliminate validations that are done twice

### PR DESCRIPTION
**Pull request by rvanheest**
fixes EASY-1672 & EASY-1756

#### When applied it will
* [x] make `SF_PLAY_MODE` mandatory
* [x] fail when `SF_PLAY_MODE = MENU` and any A/V files don't have a title declared in `FILE_TITLE`
* [x] update technical documentation
* [x] update archivist manual
* [x] eliminate validations that were done twice
* [x] code review with @janvanmansum 

@DANS-KNAW/easy for review

### CODE REVIEW

File: src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <file filepath="data/path/to/a/random/video/hubble.mpg">
         <dcterms:type>http://schema.org/VideoObject</dcterms:type>
         <dcterms:format>video/mpeg</dcterms:format>
-        <dcterms:title>hubble.mpg</dcterms:title>
+        <dcterms:title>video about the hubble space telescoop</dcterms:title>
```
Comment (janvanmansum) [inline src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml]:
```suggestion
        <dcterms:title>video about the hubble space telescope</dcterms:title>
```

File: src/test/resources/allfields/output/input-ruimtereis01/bag/tagmanifest-md5.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
 4e9f2a22c9b9ac042318e7663616c14e  metadata/dataset.xml
-d5c52d566d94c40730e421018da0b5d6  metadata/files.xml
+03502c00e9f97545720f5f1660b49875  metadata/files.xml
```
Comment (janvanmansum) [inline src/test/resources/allfields/output/input-ruimtereis01/bag/tagmanifest-md5.txt]: Will also change if you correct the spelling error.

File: src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml (hunk 1 above)
Comment (rvanheest) [reply src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml]: good catch, but given that it is in a number of places, I'll fix it manually

File: multi-deposit-instructions.md [hunk 3]
```diff
@@ -170,9 +170,8 @@ a audio or video presentation contained in the dataset:
 * The data provided in `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION` are stored for further processing
   in the `deposit.properties` file.
 * `SF_PLAY_MODE` specifies how the video's are played in Springfield. The value must either be
-  `continuous` or `menu`. If no value is specified for a dataset with video/audio, the default will
-  be set to `continuous`. This value is only allowed if `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION`
-  are provided as well.
+  `continuous` or `menu`. This value is only allowed if `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION`
+  are provided as well. When `menu` is chosen, **every A/V file** must have `FILE_TITLE` defined as well.
```
Comment (janvanmansum) [inline multi-deposit-instructions.md]:
```suggestion
  are provided as well. If `menu` is chosen, **every A/V file** must have `FILE_TITLE` defined as well.
```
Sorry for nitpicking, but I think in English this distinction is more strict than the corresponding one between "wanneer" en "als" in Dutch. See: https://dictionary.cambridge.org/us/grammar/british-grammar/if-or-when

**Review by janvanmansum — COMMENTED** [review]
It would have been better if you had made this two or more pull requests. I find this a bit hard to review. Can you add some more comments to the cover message, to indicate the key changes?

**Comment by rvanheest** [comment]
@janvanmansum aren't the individual commits and their cover messages enough? I realized this would have been better in 2 PRs, but that would lead to all sorts of merge conflicts. So I figured looking at the individual commits would get you pretty far.

**Pull request merged**